src/app/containers/ConsentBanner/Canonical/logic/README.md

```diff
@@ -0,0 +1,53 @@
+# Canonical Cookie Banner Logic
+
+```js
+import logic from './index';
```

Comment:
```suggestion
import cookies from '.';
```

Reply:
😭 Really dont like this notation but ok :P